app/assess/display_value_mappings.py

```diff
@@ -47,3 +47,12 @@
     "funding_type": ALL_VALUE,
     "status": ALL_VALUE,
 }
+
+guidance_links = {
+    "COF": "https://mhclg.sharepoint.com.mcas.ms/:w:/s/"
+    + "CommunityOwnershipFund/"
+    + "Ecv3iM7U0AtKtyHnzRrQ9dsB0HdMPvHWqAoGn1WrWM7EMA?e=6QpdUT",
+    "NSTF": "https://mhclg.sharepoint.com.mcas.ms/:w:/s/"
+    + "HomelessnessandRoughSleeping/"
+    + "EZn-Dq3eBvFDtdBqhyEZxUUBj_BP53F9TVyI0imX3NdcPw?e=PtmLwH",
+}
```

Comment:
But at the moment it’s not a script, but a code change, image build, terraform commit and PR, then deploy.

Versus a single database insertion script, which wouldn’t have any downtime from the app, also would have all fund specific logic contained in the fund store, configurable per fund.

I’m not fussed about it, just think it’s where we should be moving towards, I just see in the future when we have half a dozen funds and go to implement another, we end up having a checklist of integration points spread across repositories (this would be one) - and instead of that I’d rather it all be contained in the fund store on the db model, if that makes sense.